src/iterable-iterator/map.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Maps the values in the iterator to an array using the given function.
+ * Behaves similarly to Array.map.
+ */
+export function map<T, V>(
```

Comment:
all of these iterable utility functions don't need to be in their own files, should be consolidated into a single file. @uxder maybe can suggest a naming for it, but if we're following the current structure, maybe something like:

src/iterables/iterables.ts

then the import/usage can be something like:

```
import * as iterables from 'iterables';

iterables.map(...);
```

Reply:
Can do.

Reply:
Thanks Angus - do you think we can actually make that change against some of the recent stuff you checked in as well (can be a separate PR to avoid overloading this one).  

map-wrapper, mapped-iterator, maulti-value-array etc per trying to lower file count and keeping it all contained.

Reply:
Created issue for myself to keep current PR on track: https://github.com/grow/yano-js/issues/55